tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -719,6 +719,12 @@ internal static class Propagated
             /// lower-case hexadecimal string with no zero-padding or `0x` prefix.
             /// </summary>
             internal const string TraceIdUpper = "_dd.p.tid";
+
+            /// <summary>
+            /// Tag used to propagate the unsigned  64 bits last parent Id
+            /// lower-case 16 characters hexadecimal string
+            /// </summary>
+            internal const string LastParentId = "_dd.parent_id";
```

Comment:
I don't know that I would add this tag under `Propagated`, since it has special behavior that is different than the other `_dd.p.*` tags.

Reply:
Update in: https://github.com/DataDog/dd-trace-dotnet/pull/5721/commits/42164062f5114055ccb6a5c71c20031a6438476c